docs/source/install_developers.rst

```diff
@@ -93,31 +93,19 @@ Run tests
 For running the tests, it is required to install the development requirements. Again, first activate your
 virtualenv or conda environment.
 
-.. code::
+.. code:: bash
 
-   $ git clone --recurse-submodules https://github.com/NeurodataWithoutBorders/pynwb.git
-   $ cd pynwb
-   $ pip install -r requirements.txt -r requirements-dev.txt
-   $ pip install -e .
-   $ tox
+    git clone --recurse-submodules https://github.com/NeurodataWithoutBorders/pynwb.git
+    cd pynwb
+    pip install -r requirements.txt -r requirements-dev.txt
+    pip install -e .
+    tox
 
 For debugging it can be useful to keep the intermediate NWB files created by
 the tests. To keep these files create the environment variables
 ``CLEAN_NWB``/``CLEAN_HDMF`` and set them to ``1``.
 
 
-Following PyNWB Style Guide
```

Comment:
Removed since this section is not in the hdmf install_developers.rst

Reply:
That's fine. I think I moved it to https://hdmf.readthedocs.io/en/stable/contributing.html in HDMF. Could you update the corresponding page in PyNWB as well (if it needs an update)?

Reply:
@rly already done!